src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -724,7 +725,21 @@ const WorkflowView = _.flow(
               disabled: currentSnapRedacted || !!Utils.computeWorkspaceError(ws),
               checked: useCallCache,
               onChange: v => this.setState({ useCallCache: v })
-            }, [' Use call caching'])
+            }, [' Use call caching']),
+            span({ style: { marginLeft: '1rem', marginRight: '0.5rem' } }, [
+              h(LabeledCheckbox, {
+                checked: deleteIntermediateOutputFiles,
+                onChange: v => this.setState({ deleteIntermediateOutputFiles: v }),
+                style: { marginLeft: '1rem' }
+              }, [' Delete intermediate outputs'])
+            ]),
+            h(InfoBox, [
+              'If the workflow succeeds, only the final output will be saved. Subsequently, call caching cannot be used as the intermediate steps will be not available. ',
+              h(Link, {
+                href: 'https://support.terra.bio/hc/en-us/articles/360039681632',
+                target: 'blank'
```

Comment:
Rather than` target: 'blank' `there's a util that has been extracted out which can be applied with: `..Utils.newTabLinkProps,` which also specifies the 'rel' attribute when opening a link

Reply:
changed — thanks!